software/winrm/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -72,8 +72,10 @@
             "address",
             "Address of the remote machine");
 
-    public static final ConfigKey<Integer> WINRM_PORT = WinRmTool.PROP_PORT;
-    
+    public static final ConfigKey<Integer> WINRM_PORT = WinRmTool.PROP_PORT.getConfigKey();
+    public static final ConfigKey<Boolean> USE_SECURE_WINRM = WinRmTool.USE_SECURE_WINRM;
+    public static final ConfigKey<Boolean> USE_NTLM = WinRmTool.USE_NTLM;
```

Comment:
In an ideal world this would be a list of white listed authentication protocols - 'Kerberos, NTLM' by default. But good enough for this PR - could mark it `@Beta`, subject to change.